cluster/region/region_test.go

```diff
@@ -121,3 +121,42 @@ func TestRegion_GetSize(t *testing.T) {
 	size := region.GetSize()
 	assert.Equal(t, int64(100), size)
 }
+
+func TestRegion_AddPeer(t *testing.T) {
```

Comment:
As for how to test, you can leave it for the time being. It need to really start the raft cluster to test, and the code for starting the raft has not been written yet.